src/packagedcode/models.py

```diff
@@ -1530,13 +1530,22 @@ class Package(PackageData):
         label='datasource ids',
         help='List of the datasource ids used to create this package.'
     )
-
+    
+    license_clarity_score = List(
+        item_type=dict,
```

Comment:
This is a dict, so why do you have this in a List function? Does this even work?

Reply:
yes it was working, but initially I was thinking in another direction, but now I think we can have this in `dict` only.
One of the test results had this List of the License Clarity Score
<img width="1440" alt="Screenshot 2024-06-26 at 4 47 59 PM" src="https://github.com/nexB/scancode-toolkit/assets/81990329/3956bbf8-b6e3-417c-83c9-c0115950938d">
but in the next commit, I will change it to dict only.